tensorflow_probability/python/bijectors/conditional_bijector_test.py

```diff
@@ -18,6 +18,8 @@
 from __future__ import division
 from __future__ import print_function
 
+import collections
```

Comment:
Just one last thing, this is now unused.

Reply:
Good catch